src/reservation/reservation.service.ts

```diff
@@ -15,4 +15,34 @@ export class ReservationService {
       },
     });
   }
+
+  async getReservations(id: string) {
+    return await this.databaseService.user.findUnique({
```

Comment:
não seria mais interessante fazer a requisição pelo `this.databaseService.reservation` invés do de usuário?